generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs

```diff
@@ -2488,9 +2504,11 @@
     "clientSessionMaxLifespan": "0",
     "parRequestUriLifespan": "60",
     "clientOfflineSessionIdleTimeout": "0",
-    "cibaInterval": "5"
+    "cibaInterval": "5",
+    "frontendUrl": "",
+    "acr.loa.map": "[]"
   },
-  "keycloakVersion": "<%= DOCKER_KEYCLOAK_VERSION %>",
+  "keycloakVersion": "19.0.1",
```

Comment:
```suggestion
  "keycloakVersion": "<%= DOCKER_KEYCLOAK_VERSION %>",
```

Reply:
doesn't really matter as each upgrade requires to review the realm configurations. Having a constant based substitution doesn't provide an insight into that dependency. I would rather prefer to set an explicit version.

Reply:
I'm OK with either way. Chances are I can Ctrl+Click in my IDE on the variable and see the value.

Reply:
@DanielFran, If you have a strong opinion on this change, then, feel free to commit the suggestion. Otherwise, tests looks good for changes to be reviewed.